Modules/Core/Common/include/itkSymmetricEigenAnalysis.h

```diff
@@ -756,15 +756,12 @@ class ITK_TEMPLATE_EXPORT SymmetricEigenAnalysisFixedDimension
     else   { m_OrderEigenValues = DoNotOrder;       }
   }
   bool GetOrderEigenMagnitudes() const { return ( m_OrderEigenValues == OrderByMagnitude ); }
-  constexpr unsigned int GetOrder() const { return m_Order; }
-  constexpr unsigned int GetDimension() const { return m_Dimension; }
-  constexpr bool GetUseEigenLibrary() const { return m_UseEigenLibrary; }
+  constexpr unsigned int GetOrder() const { return TMatrixDimension; }
```

Comment:
I wonder if the usage of a constexpr variable in a constexpr member function might be the different usage causing this linkage problem.

Reply:
I think it was coming from some function call where this was passed as a parameter, maybe [this](https://github.com/InsightSoftwareConsortium/ITK/pull/248/commits/4866a116041afe46f94775a21c163fbeba9097f0#diff-4506b39b12c5fd4295bd9485f7ac8745R886).